src/miner.cpp

```diff
@@ -92,7 +92,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
 CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& scriptPubKeyIn,bool blockstreamCoreCompatible)
 {
     // Create new block
-    unique_ptr<CBlockTemplate> pblocktemplate(new CBlockTemplate());
```

Comment:
Really I just rebased care's work. Did you do this manually ? If so, you should consider adding this to the pile of wasted effort due to lack of CI.

Generally, importing the whole std namespace is considered bad practice so I'd go for core style here.